blazor/dropdown-list/templates.md

```diff
@@ -7,307 +7,107 @@ control: DropDown List
-    </DropDownListTemplates>
-    <SfDataManager Url="https://ej2services.syncfusion.com/production/web-services/api/Employees" Adaptor="Syncfusion.Blazor.Adaptors.WebApiAdaptor" CrossDomain="true"></SfDataManager>
-    <DropDownListFieldSettings Text="FirstName" Value="Designation"></DropDownListFieldSettings>
-</SfDropDownList>
-
-@code {
-    public class EmployeeData
-    {
-        public string FirstName { get; set; }
-        public string Designation { get; set; }
-    }
-    public Query Query = new Query();
-}
-
-<style>
-    .destination {
-        right: 15px;
-        position: absolute;
-    }
-</style>
-```
-
-
-
-![Blazor DropDownList with ValueTemplate](./images/blazor-dropdownlist-value-template.png)
+## Item template
 
-## Group template
+The content of each list item within the DropDownList can be customized with the help of [ItemTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_ItemTemplate) property.
 
-The group header title under which appropriate sub-items are categorized can also be customized with the help of [GroupTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_GroupTemplate) property. This template is common for both inline and floating group header template.
+In the following sample, each list item is split into two columns to display relevant data.
 
-In the following sample, employees are grouped according to their city.
+{% highlight cshtml %}
 
-```cshtml
-@using Syncfusion.Blazor.Data
-@using Syncfusion.Blazor.DropDowns
+{% include_relative code-snippet/template/item-template.razor %}
 
-<SfDropDownList TValue="string" TItem="EmployeeData" Placeholder="Select a customer" Query="@Query">
-    <DropDownListTemplates TItem="EmployeeData">
-        <GroupTemplate>
-            <span class="group">@(context.Text)</span>
-        </GroupTemplate>
-    </DropDownListTemplates>
-    <SfDataManager Url="https://ej2services.syncfusion.com/production/web-services/api/Employees" Adaptor="Syncfusion.Blazor.Adaptors.WebApiAdaptor" CrossDomain=true></SfDataManager>
-    <DropDownListFieldSettings Value="FirstName" GroupBy="Country"></DropDownListFieldSettings>
-</SfDropDownList>
+{% endhighlight %}
 
-@code {
+![Blazor DropDownList with ItemTemplate](./images/template/blazor-dropdownlist-item-template.png)
 
-    public class EmployeeData
-    {
-        public string FirstName { get; set; }
-        public string Country { get; set; }
-    }
+## Group template
+
+The group header title under which appropriate sub-items are categorized can also be customized with the help of [GroupTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_GroupTemplate) property. This template is common for both inline and floating group header template.
```

Comment:
The group header title under which appropriate sub-items are categorized can also be customized with the help of the [GroupTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_GroupTemplate) property. This template is common for both inline and floating group header template.